client-projects/the-lumery/index.ts

```diff
@@ -44,6 +46,13 @@ document.addEventListener("DOMContentLoaded", () => {
     setUpFilterButtons();
   }
 
+  // images within cards fade up and on when they appear on screen, and have a
+  // hover effect applied.
+  if (document.querySelectorAll(CARD_LINK_SELECTOR)) {
```

Comment:
I think this will always be truthy, even if no items are found. You might need to specifically check for the `length`?

Reply:
ah yep you're right